CONTRIBUTING.md

```diff
@@ -97,7 +97,9 @@ If a fix for the vulnerability isn't available or if it isn't practical to uptak
 This repository uses [Chromatic](https://www.chromatic.com) to facilitate visual component review, and adds GitHub status checks to the build pipeline. The workflow is as follows:
 
 1. The `UI Tests` status check is designed to highlight any visual changes included in the changeset. The developer (that's you!) should review the `UI Tests` status check in Chromatic, and if all changes are intentional or expected, mark the components as **approved**.
-2. The `UI Review` status check is designed to collect feedback from UX and visual designers. Using the Chromatic review tooling, invite designers to review and approve the component changes.
+2. The `UI Review` status check is designed to collect feedback from UX and visual designers. Rather than blocking PR completion on this feedback, you can also approve this check. However, you should still demo your changes to relevant designers either in a team meeting or one-on-one. This can happen either before or after the PR completes, as long as designer feedback is addressed promptly.
```

Comment:
```suggestion
2. The `UI Review` status check is designed to collect feedback from UX and visual designers. Rather than blocking PR completion on this feedback, you can also approve this check by validating the story changes yourself. However, you should still demo your changes to relevant designers either in a team meeting or one-on-one. This can happen either before or after the PR completes, as long as designer feedback is addressed promptly.
```

> However, you should still demo your changes to relevant designers either in a team meeting or one-on-one.

How does this apply to external contributors? I'd guess the nimble team approvers should be responsible for that where appropriate (depending on how external the contributor is)? Ultimately a nimble CODEOWNER has to approve it so maybe it's not a big deal to define right now